site/docs/messaging/campaign-management/about.mdx

```diff
@@ -23,5 +23,5 @@ export const Svg = require('../../../static/img/icon-messaging.svg').default;
 
 | Guide | Description |
 |:------|:------------|
-|[10DLC CSP Campaigns](/docs/messaging/campaign-management/csp) | How to provision and manage 10DLC campaigns via Bandwidth CSP Portal |
+|[10DLC Direct Customer Campaigns](/docs/messaging/campaign-management/csp) | How to provision and manage 10DLC campaigns via Bandwidth CSP Portal |
```

Comment:
May want to change this to just 10DLC Campaigns or something, Direct Customer may be a little misleading since they can register as Reseller or Direct Customer